addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover_tests.js

```diff
@@ -241,6 +251,44 @@ QUnit.test('activity mark done popover mark done and schedule next', async funct
     );
 });
 
+QUnit.test('activity mark done with new action', async function (assert) {
```

Comment:
Alex will probably say something about this test name but I don't have a suggestion right now.

Reply:
At a glance, it asserts something when clicking on mark as done & schedule next of an activity.
The action "mark done & schedule next" should be visible in test name.

As for the test itself and its intent, I have mixed feeling. Mocking RPC result is something we highly avoid to do, because this makes test too technical. The intercept of `do-action` is an unfortunate lack of UI response in context of test with action manager, which is not conveniently testable enough to say the least.

All this to say that due to how lacking mock server and `do-action` management in test environment, the assertions resulting from "mark as done and schedule next" looks very artificial.

I don't like such tests, so if we want to keep this test with full knowledge of problems it has, please put `[technical]` tag in the test name, and comment what are the technical details expected following the click on "Mark as done & Schedule next".

This is very important to do that in my eyes, because current test is a nightmare to understand/debug/maintain as it is. Flag `[technical]` + comment helps understanding why this test is/was useful, and also let us know test is prone to annoyances due to being "badly" written (quotes because we can't do much better right now...).